src/components/stack-chart/index.js

```diff
@@ -238,7 +238,7 @@ class StackChartImpl extends React.PureComponent<Props> {
         role="tabpanel"
         aria-labelledby="stack-chart-tab-button"
       >
-        <StackSettings />
+        <StackSettings hideInvertCallstack={true} />
```

Comment:
FTR I also looked at an inverted tree in the stack chart recently and also found this was useless. Thanks for the change!